specification/schemas/Shipment.json

```diff
@@ -10,8 +10,10 @@
         "relationships"
       ],
       "properties": {
+        "id": {
+          "readOnly": true
+        },
         "attributes": {
-          "type": "object",
```

Comment:
Should this be removed?

Reply:
Nope, that was leftover from when I tried to extend the `BaseShipment` schema. I'll add it back.